libImaging/QuantPngQuant.c

```diff
@@ -0,0 +1,141 @@
+/* Copyright (c) 2010 Oliver Tonnhofer <olt@bogosoft.com>, Omniscale
+//
```

Comment:
There's no original, I copy pasted the header from another file, probably one of `quant*.c`. Sorry.